src/components/UI/Login/Login.jsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useState, useContext } from "react";
+
+import { authContext } from "../../../contexts/AuthContext/AuthContext";
+import "./Login.css";
+const Login = () => {
+  const [showPassword, setShowPassword] = useState(false);
+  const [userData, setUserData] = useState({
+    email: "",
+    password: "",
+  });
+
+  const { authChecker, error, setError } = useContext(authContext);
+
+  const formLoginHandler = async (e) => {
+    e.preventDefault();
+
+    authChecker(userData);
+  };
+
+  const showPasswordHandler = (e) => {
+    setShowPassword(!showPassword);
+  };
+  return (
+    <div className="login-container">
+      <div className="banner"></div>
+      <form className="login-form" onSubmit={formLoginHandler}>
+        <div
+          className={`alert alert-danger mt-1  flex-justify-center flex-align-item-center ${
+            !error ? "hide" : "block"
+          }`}
+        >
+          Account Credentails Didnt Matched
+        </div>
+        <div className="flex flex-column gap ui-card-default py-1 card-my-1 pb-2 bg-form">
+          <div className="flex flex-align-item-center flex-justify-center pb-2 pt-2">
+            <h4 className="font-sm label-text-primary">Welcome To Zeplin</h4>
+          </div>
+          <div className="flex flex-align-item-center flex-justify-center pb-2 pt-2">
+            <h4 className="font-sm label-text-primary">Login</h4>
+          </div>
+          <div className="flex flex-column gap-1">
+            <label
+              htmlFor="emailInput1"
+              className="label-text label-text-primary"
+            >
+              Email Address
+            </label>
+            <input
+              type="email"
+              className="input"
+              placeholder="Email"
+              id="emailInput1"
+              onChange={(e) =>
+                setUserData({ ...userData, email: e.target.value })
+              }
+              aria-describedby="emailHelp"
+              pattern="^[a-z0-9](\.?[a-z0-9]){5,}@g(oogle)?mail\.com$"
```

Comment:
This is new to me. So we can use the pattern attribute to validate user input. Great work.